.travis.yml

```diff
@@ -0,0 +1,31 @@
+sudo: required
+
+language: scala
+
+services:
+- docker
+
+cache:
+  directories:
+  - $HOME/.ivy2/cache
+  - $HOME/.sbt/boot/
+
+script:
+  - /bin/true
```

Comment:
I copied that from mmw-geoprocessing. I think it's because we don't have any unit tests, so there's no point building it. But maybe it's worth building just to ensure there are no syntax errors? /cc @hectcastro